intro-Linux/ch_common_pitfalls.tex

```diff
@@ -103,36 +103,37 @@ \subsubsection{Copying files with WinSCP}
 \subsubsection{Permissions}
 
 \begin{prompt}
-%\shellcmd{ls -l $<$file$>$ \# File with correct permissions}%
--rwxr-xr-\strong{x} 1 %\userid% %\userid% 2983 Jan 30 09:13 $<$file$>$
-%\shellcmd{ls -l $<$file$>$ \# File without correct permissions}%
--rw-r--r-\strong{-} 1 %\userid% %\userid% 2983 Jan 30 09:13 $<$file$>$
+%\shellcmd{ls -l script.sh \# File with correct permissions}%
+-rwxr-xr-x 1 %\userid% %\userid% 2983 Jan 30 09:13 script.sh
+%\shellcmd{ls -l script.sh \# File without correct permissions}%
+-rw-r--r-- 1 %\userid% %\userid% 2983 Jan 30 09:13 script.sh
```

Comment:
I would explain this a bit better... `without correct permissions` should also be `incorrect permissions` imho